tests/test_model.py

```diff
@@ -1090,9 +1187,11 @@ def test_loading_and_saving_hybrid_model(movielens_metadata_df, train_val_implic
     # ``get_item_predictions`` is the same
     loaded_model.set_stage('matrix_factorization')
 
-    actual = loaded_model.get_item_predictions(user_id=42, unseen_items_only=False)
+    item_preds_actual = loaded_model.get_item_predictions(user_id=42, unseen_items_only=False)
```

Comment:
Can we also update the comment a bit above this, L1187, to include the user method too?